components/Footer.js

```diff
@@ -19,6 +19,9 @@ const Footer = () => (
       <a className="link" href="https://github.com/carbon-app/carbon">
         source
       </a>
+      <a className="link" href="/contributors">
```

Comment:
I'm not sure I want to add it to the footer yet. Can we add it to the "About" page?

Reply:
Yeah for sure, let me add it under the about page!